pkg/infra/server/baseServer.go

```diff
@@ -0,0 +1,77 @@
+package infra
+
+import (
+	"fmt"
+	"strconv"
+	"github.com/google/uuid"
+)
+
+type BaseServer struct {
+	NumAgents int
+	NumTurns  int
+	Agents    []Agent
+}
+
+type Agent struct {
+	ID uuid.UUID
+	name string
+	floor int
+	energy int
```

Comment:
This is too specific - I think that this came from a previous Cohort. I'd remove them to keep the implementation general